libcloudforensics/providers/gcp/internal/common.py

```diff
@@ -187,3 +187,45 @@ def BlockOperation(self, response, zone=None):
       if result['status'] == 'DONE':
         return result
       time.sleep(5)  # Seconds between requests
+
+
+def ExecuteAndPaginate(client, func, kwargs, throttle=False):
```

Comment:
Why is this called `ExecuteAndPaginate`? Even if it indeed uses pagination under the hood, does it really change anything for the caller? It doesn't seem like the caller can specify page sizes or a given page number.

Reply:
I have changed it to `ExecuteRequest`